be-a-managing-director/readme.md

```diff
@@ -0,0 +1,66 @@
+# How to be a Managing Director
+
+Great, so you’re the Managing Director of New Atoms, congratulations🎉!
+
+Lets take apart what that means for a second, in reverse order:
+
+1. Managing: Having executive control. Meaning you are the person that needs to deal with shit.
```

Comment:
I think its more like this, Manging; making sure everything goes as planned and towards the right direction, meaning you have to deal with all the shit that is in the way